payment-processor/src/main/kotlin/org/ostelco/prime/paymentprocessor/StripeUtils.kt

```diff
@@ -23,29 +24,33 @@ object StripeUtils {
         return try {
             Either.right(action())
         } catch (e: CardException) {
-            // If something is decline with a card purchase, CardException will be caught
-            logger.warn("Payment error : $errorDescription , Stripe Error Code: ${e.code}")
-            Either.left(ForbiddenError(errorDescription, e.message))
+            /* Something is wrong with the payment source/card.
+               402 indicates that the card is out of funds or expired etc. */
+            logger.warn("Payment error: ${errorDescription}, Stripe Error Code: ${e.code}")
+            Either.left(if (e.statusCode == 402)
```

Comment:
Please don't use naked numbers.   Find a constant that represents the semantics of this particular return code (I didn't remember  off hand that 402 means "payment required", something that would have been obvious if a descriptive enumeration name had been used instead). There are plenty of them in the various HTTP utility classes we use, please use one of them.